src/Extension/OrderedButtonExtension.php

```diff
@@ -24,6 +24,6 @@ class OrderedButtonExtension extends AbstractOrderedExtension
     */
     public function getExtendedType()
     {
-        return 'button';
+        return 'Symfony\Component\Form\Extension\Core\Type\ButtonType';
```

Comment:
@matthiasnoback Unfortunately, I'm afraid that there's no way to do one or another depending on the version number. For instance, [Symfony developpers recommands to implement both methods](https://github.com/symfony/symfony/blob/v2.8.0-BETA1/UPGRADE-2.8.md#form) about `getName()` deprecation:

> If you want to create types that are compatible with Symfony 2.3 up to 2.8 and don't trigger deprecation errors, implement both getName() and getBlockPrefix()